sqlserver/tests/test_unit.py

```diff
@@ -735,3 +738,67 @@ def test_extract_sql_comments_and_procedure_name(query, expected_comments, is_pr
     assert comments == expected_comments
     assert p == is_proc
     assert re.match(name, expected_name, re.IGNORECASE) if expected_name else expected_name == name
+
+
+class DummyLogger:
+    def debug(*args):
+        pass
+
+    def error(*args):
+        pass
+
+
+def set_up_submitter_unit_test():
```

Comment:
is there some more unit testing that can be added? We don't seem to test any of the error paths

Reply:
I'll try to add more tests today after I test backend

Reply:
added some tests pls take a look